### PR TITLE
zh-translation1.12.x Releases

### PR DESCRIPTION
**Pull request by yanrongshi**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/news/releases/1.12.x/_index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
```
Comment (kebe7jun) [inline content/zh/news/releases/1.12.x/_index.md:1]: It sense that you deleted old content.

Comment (yanrongshi) [reply content/zh/news/releases/1.12.x/_index.md:1]: Yeah, maybe I didn't notice. It's been corrected

**Comment by yanrongshi** [comment]
Good afternoon,dear reviewer, What should I do next?

**Comment by yanrongshi** [comment]
Good afternoon,dear reviewer, What should I do next?

File: content/zh/news/releases/1.12.x/_index.md (hunk 1 above)
Comment (kebe7jun) [reply content/zh/news/releases/1.12.x/_index.md:1]: you didn't fix this.

**Review by kebe7jun — APPROVED** [review]

**Pull request merged**